src/app/page.tsx

```diff
@@ -186,6 +187,14 @@ export default function Home() {
           View All
         </Button>
       </PageSection>
+      <CTASection
+        title="Become Part of Our Vibrant&nbsp;Community"
```

Comment:
Why are we adding `&nbsp` - Do we need to break this intentionally? Asking because it effects responsiveness.

Reply:
I added it to avoid having [orphan text](https://gomakethings.com/a-css-native-way-to-prevent-orphan-text/) - poor "Community" hanging by itself. Could remove if you're not a fan

Reply:
This is a good idea. I tried `text-wrap: pretty` but it didn't seem to do much. `text-wrap: balance` did work - which was a suggestion that @CharlyMartin made earlier. I've now added as a default `baseStyle` to the `Heading` component. Please do keep in mind that it's not fully supported yet [>>](https://caniuse.com/css-text-wrap-balance)